chef/cookbooks/neutron/recipes/server.rb

```diff
@@ -149,6 +149,9 @@
 num_vlans = node[:neutron][:num_vlans]
 vlan_end = [vlan_start + num_vlans - 1, 4094].min
 
+vni_start = [node[:neutron][:vxlan][:vni_start], 0].max
+vni_end = [node[:neutron][:vxlan][:vni_end], 16777215].min
```

Comment:
Hmm, I don't remember our previous discussions but I can't really agree in this case.  If I changed a value via Chef and accidentally (or through misunderstanding of the attribute) provided an invalid value, I would much prefer to be told about it then have that value silently ignored, because the latter behaviour is more likely to result in confusion from unexpected side-effects later on.  And noone except developers should be changing values via Chef anyway.  But if that doesn't persuade you then I guess let's just agree to disagree and go with your version; it's not a big deal.